modules/local/buildconfig/main.nf

```diff
@@ -0,0 +1,93 @@
+import groovy.json.JsonSlurper
+import groovy.json.JsonOutput
+
+process BUILD_CONFIG {
+    tag "Build Arborator Config"
+    label 'process_single'
+
+    input:
+    val metadata_headers
+    // tuple(ID (header name), metadata_partition_name, metadata_1_header, .. , metadata_8_header)
```

Comment:
Could you either move this on the same line as `metadata_headers` or above that line? I thought at first this was another input parameter that was commented out

Reply:
Sure thing: f70f439436e0fcb6816e28c31871f0c084e53b91